teams_accounts.go

```diff
@@ -227,6 +237,26 @@ func (api *API) TeamsAccountLoggingConfiguration(ctx context.Context, accountID
 	return teamsConfigResponse.Result, nil
 }
 
+// TeamsAccountConnectivityConfiguration returns zero trust account connectivity settings.
+//
+// API reference: https://developers.cloudflare.com/api/operations/zero-trust-accounts-get-connectivity-settings
```

Comment:
```suggestion
// API reference: https://developers.cloudflare.com/api/operations/zero-trust-accounts-get-connectivity-settings
```

Reply:
Thanks for reviewing @jacobbednarz, I'm not seeing a diff in the suggested change.

Reply:
unsure what is happening here :/ it has the right URL so 🤷